src/Test/Unit/Step/PostDeploy/WarmUpTest.php

```diff
@@ -84,9 +92,13 @@ public function testExecute()
             'http://base-url.com/index.php',
             'http://base-url.com/index.php/customer/account/create'
         ];
+        $concurrency = 0;
```

Comment:
Please cover the case when concurrency is different than 0.

Reply:
Done